pom.xml

```diff
@@ -45,6 +45,10 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
+        <!--@todo #909:30min jcabi-parent used by Netbout is pretty outdated,
```

Comment:
@dmzaytsev This doesn't fit into the ticket description, better to issue a new ticket against it (and get paid separately ;) )